File: Progra4_project/src/java/AAMAirline/model/Ciudad.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package AAMAirline.model;

/**
 *
 * @author jimen
 */
public class Ciudad implements Jsonable{

    public Ciudad(String codigo_ciudad, String nombre, String pais) {
        this.codigo_ciudad = codigo_ciudad;
        this.nombre = nombre;
        this.pais = pais;
    }

    public String getCodigo_ciudad() {
        return codigo_ciudad;
    }

    public void setCodigo_ciudad(String codigo_ciudad) {
        this.codigo_ciudad = codigo_ciudad;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getPais() {
        return pais;
    }

    public void setPais(String pais) {
        this.pais = pais;
    }


 
    private String codigo_ciudad;
    private String nombre;
    private String pais;
}

File: Progra4_project/web/js/Ciudad.js
 // funcion Ciudad
function Ciudad (codigo_Ciudad,nombre,pais){ 
  this.Ciudad(codigo_Ciudad,nombre,pais);
}

// solo metodos
Ciudad.prototype={ 
  Ciudad: function(codigo_Ciudad,nombre,pais){
      this.codigo_Ciudad=codigo_Ciudad;
      this.nombre=nombre;
      this.pais=pais;
  }
};